mapilio_kit/base/video_upload.py

```diff
@@ -10,10 +12,16 @@ def fundamental_arguments(self, parser: argparse.ArgumentParser):
         uploader().fundamental_arguments(parser)
 
     def perform_task(self, vars_args: dict):
-        from . import uploader,sampler
+        from . import uploader, sampler
+        video_paths = os.listdir(vars_args["video_import_path"]) if os.path.isdir(vars_args["video_import_path"]) else [vars_args["video_import_path"]]
+        video_formats = ('.mp4', '.avi', '.mov', '.mkv', '.360', '.MP4', '.AVI', '.MOV', '.MKV')
 
-        if not vars_args['processed']:
-            sampler().perform_task(vars_args)
-        else:
-            vars_args["import_path"]=vars_args["video_import_path"]
-        uploader().perform_task(vars_args)
+        vars_args_copy=deepcopy(vars_args)
```

Comment:
Why using deepcopy ?

Reply:
Because some arguments needs to be overwritten